var/spack/repos/builtin/packages/libelf/package.py

```diff
@@ -38,13 +38,11 @@ class Libelf(Package):
     version('0.8.12', 'e21f8273d9f5f6d43a59878dc274fec7')
 
     provides('elf')
+    depends_on('automake')
 
-    def install(self, spec, prefix):
-        configure("--prefix=" + prefix,
-                  "--enable-shared",
-                  "--disable-dependency-tracking",
-                  "--disable-debug")
-        make()
-
-        # The mkdir commands in libelf's install can fail in parallel
-        make("install", parallel=False)
```

Comment:
it worked OK for me, should I put it back in case it is still broken for others?